README.md

```diff
@@ -1,318 +1,55 @@
-
-    | Option | Description |
-    | ------ | ----------- |
-    | **bucket** | The name of the AWS S3 bucket to use |
-    | **apiVersion** | Version of the S3 API. Example: `'2006-03-01'` |
-    | **endpoint** | When using a local or proxy S3 instance, set this value to the host URL. Example: `http://localhost:9000` |
-    | **s3ForcePathStyle** | Set to `true` if using a local or proxy S3 instance |
-
-3. Copy [api/v1/db/awsS3/pets-dao-example.js](api/v1/db/awsS3/pets-dao-example.js) to `api/v1/db/awsS3/<resources>-dao.js` and modify as necessary:
-
-    ```shell
-    $ cp api/v1/db/awsS3/pets-dao-example.js api/v1/db/awsS3/<resources>-dao.js
-    ```
-
-4. Make sure to require the correct path for the new DAO file at path handlers files:
-
-    ```js
-    const petsDao = require('../db/awsS3/<resources>-dao');
-    ```
-
-## Docker
-
-[Dockerfile](Dockerfile) is also provided. To run the app in a container, install [Docker](https://www.docker.com/) first, then:
-
-1. Modify `WORKDIR` from the [Dockerfile](Dockerfile#L4-L5):
-
-    ```Dockerfile
-    # Copy folder to workspace
-    WORKDIR /usr/src/<my-api>
-    COPY . /usr/src/<my-api>
-    ```
 
-2. If the API requires [node-oracledb](https://oracle.github.io/node-oracledb/) to connect to an Oracle database, download an [Oracle Instant Client 12.2 Basic Light zip (64 bits)](http://www.oracle.com/technetwork/topics/linuxx86-64soft-092277.html) and place into `./bin` folder. In addition, uncomment [the following code](Dockerfile#L11-L18) from the Dockerfile:
+ ##  PUT
 
-    ```Dockerfile
-    # Install Oracle Instant Client
-    RUN apt-get update && apt-get install -y libaio1 unzip
-    RUN mkdir -p /opt/oracle
-    RUN unzip bin/instantclient-basiclite-linux.x64-12.2.0.1.0.zip -d /opt/oracle
-    RUN cd /opt/oracle/instantclient_12_2 \
-        && ln -s libclntsh.so.12.1 libclntsh.so \
-        && ln -s libocci.so.12.1 libocci.so
-    RUN echo /opt/oracle/instantclient_12_2 > /etc/ld.so.conf.d/oracle-instantclient.conf \
-        && ldconfig
-    ```
+ ### /games/{id}
+Update the game by changing its round status. Other information of the game are not supposed to be changed by this endpoint.
 
-3. Build the docker image:
+ ### /players/{id}
+Update the player by changing its bet or status. Other information of the player are not supposed to be changed by this endpoint.
 
-    ```shell
-    $ docker build -t <my-api> .
-    ```
+ ### DELETE
 
-4. Run the app in a container:
+ ### /games/{id}
+Delete a game with its id.
 
-    ```shell
-    $ docker run -d \
-                 -p 8080:8080 \
-                 -p 8081:8081 \
-                 -v path/to/keytools/:/usr/src/<my-api>/keytools:ro \
-                 -v "$PWD"/config:/usr/src/<my-api>/config:ro \
-                 -v "$PWD"/logs:/usr/src/<my-api>/logs \
-                 --name <my-api> \
-                 <my-api>
-    ```
+ ### /players/{id}
+Delete a player from a certain game(player leaving the game).
```

Comment:
Since we've used OpenAPI/Swagger file to define the API specification and behavior, there is no need to repeat them again in your README. However, we do wanna include how to install, run and test all kinds of these instruction steps on our README file. You can still keep the description and purpose parts in this file, but I'd suggest pulling the rest part from the skeleton again.

Reply:
Did you address this?